### PR TITLE
Create bilibili

### PR DESCRIPTION
**Pull request by rhjdvsgsgks**

### CODE REVIEW

File: data/bilibili [hunk 1]
```diff
@@ -0,0 +1,8 @@
+bilibili.com
+www.im9.com
+acg.tv
+hdslb.com
+dataflow.biliapi.com
```
Comment (VictoriaRaymond) [inline data/bilibili]: `biliapi.com` is find. `dataflow.` is not necessary.

File: data/bilibili [hunk 2]
```diff
@@ -0,0 +1,8 @@
+bilibili.com
+www.im9.com
```
Comment (VictoriaRaymond) [inline data/bilibili]: this site is no longer active.

Comment (rhjdvsgsgks) [reply data/bilibili]: But these domains was provided by official application
screenshot

Comment (VictoriaRaymond) [reply data/bilibili]: Ok. Please remove `www.` at beginning.

Comment (rhjdvsgsgks) [reply data/bilibili]: Like this?

**Pull request merged**